Geometry/CMSCommonData/python/Phase1_R34F16_cmsSimIdealGeometryXML_cfi.py

```diff
@@ -232,7 +236,7 @@
         'Geometry/ForwardCommonData/data/totemRotations.xml', 
         'Geometry/ForwardCommonData/data/totemt1.xml', 
         'Geometry/ForwardCommonData/data/totemt2.xml', 
-        'Geometry/ForwardCommonData/data/ionpump.xml', 
+        'Geometry/ForwardCommonData/data/ionpump.xml')+cms.vstring( 
```

Comment:
The list is too long
________________________________
From: David Lange [notifications@github.com]
Sent: 09 July 2015 21:37
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx20 Update the scenarios by adding relevant xml files (#10100)


In Geometry/CMSCommonData/python/Phase1_R34F16_cmsSimIdealGeometryXML_cfi.py<https://github.com/cms-sw/cmssw/pull/10100#discussion_r34294339>:

> @@ -232,7 +236,7 @@
>          'Geometry/ForwardCommonData/data/totemRotations.xml',
>          'Geometry/ForwardCommonData/data/totemt1.xml',
>          'Geometry/ForwardCommonData/data/totemt2.xml',
> -        'Geometry/ForwardCommonData/data/ionpump.xml',
> +        'Geometry/ForwardCommonData/data/ionpump.xml')+cms.vstring(


@bsunanda<https://github.com/bsunanda> - why the cms.vstring() is needed?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/10100/files#r34294339>.